src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -98,7 +98,8 @@ private void addRandomUsers() {
     Collections.shuffle(randomUsernames);
 
     for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
-      User user = new User(UUID.randomUUID(), randomUsernames.get(i), Instant.now());
+      User user = new User(UUID.randomUUID(), randomUsernames.get(i), Instant.now(), 
```

Comment:
I think if the parameters can't all fit on one line, it's more readable if you make each parameter on its own line!

Reply:
Generally, we don't do this, except in initializing data. I would just keep to 80char width at least, and wrap when it reaches that.

However, the wrapped line should be vertically aligned with the open paren after User or all on the next line.

Example:
```
User user = new User(UUID.randomUUID(), randomUsernames.get(i), Instant.now(),
                     randomUsernames.get(i));
```

or:
```
User user = new User(
    UUID.randomUUID(), randomUsernames.get(i), Instant.now(), randomUsernames.get(i));
```

or if that doesn't fit:
```

or:
```
User user = new User(
    UUID.randomUUID(), randomUsernames.get(i), Instant.now(),
    randomUsernames.get(i));
```